src/jobservice/runner/redis.go

```diff
@@ -94,7 +103,7 @@ func (rj *RedisJob) Run(j *work.Job) (err error) {
 
 		// Log error and exit
 		logger.Errorf("Job '%s:%s' exit with error: failed to get job tracker: %s", j.Name, j.ID, err)
-
+		tracelib.RecordError(span, err, "failed to get job tracker")
```

Comment:
do we need to track the id, name... ?

Reply:
Yes, if no job name, user will not know what kind of job they are tracing. And job id used to identify job, if user wanna know the details of job, they can use these id to check job log

Reply:
do you mean, we need to add job id and name in this line?
no, we didn't, because the line previous code 
```
	span.SetAttributes(attribute.Key("jobID").String(jID), attribute.Key("jobName").String(j.Name))
```
already set those attribute